winterfell/webapp/src/js/headerCtrl.js

```diff
@@ -32,19 +32,17 @@ app.controller("headerCtrl",
 
         }, function failure(res) {
           that.active = true;
-          var modal = $uibModal.open({templateUrl: 'templates/modals/sessionExpired.html', windowClass: 'ngdialog-theme-default'});
-          modal.result.then(
-            function success() {
-              console.log("success");
-              that.active = false;
-              $window.location.href = '/login';
-            },
-            function failure() {
-              console.log("fail");
-              that.active = false;
-              $window.location.href = '/login';
-            }
-          );
+          $scope.headline = "Session Expired";
+          $scope.message = "For security reasons, your session has expired due to inactivity. Please log back in to continue your work.";
+          var modal = $uibModal.open({
+            scope: $scope,
```

Comment:
Instead of polluting the parent scope, might be cleaner to pass `scope: {headline: ..., message: ...}`